### PR TITLE
Sync Payment Methods between Host Dashboard and Expenses Page

### PR DESCRIPTION
**Pull request by SudharakaP**
Resolve https://github.com/opencollective/opencollective/issues/4453

Related to https://github.com/opencollective/opencollective-frontend/pull/6625

test-expense-2

### CODE REVIEW

File: components/collective-page/sections/Budget.js [hunk 1]
```diff
@@ -24,8 +24,9 @@ import TransactionItem from '../../transactions/TransactionItem';
 import { withUser } from '../../UserProvider';
 import BudgetStats from '../BudgetStats';
 import ContainerSectionContent from '../ContainerSectionContent';
+import { budgetSectionQuery } from '../graphql/queries';
 
-export const budgetSectionQuery = gqlV2/* GraphQL */ `
+export const budgetSectionQueryWithHost = gqlV2/* GraphQL */ `
   query BudgetSection($slug: String!, $hostSlug: String!, $limit: Int!, $kind: [TransactionKind]) {
```
Comment (znarf) [inline components/collective-page/sections/Budget.js:30]:
```suggestion
  query BudgetSectionWithHost($slug: String!, $hostSlug: String!, $limit: Int!, $kind: [TransactionKind]) {
```

File: components/collective-page/sections/Budget.js [hunk 2]
```diff
@@ -24,8 +24,9 @@ import TransactionItem from '../../transactions/TransactionItem';
 import { withUser } from '../../UserProvider';
 import BudgetStats from '../BudgetStats';
 import ContainerSectionContent from '../ContainerSectionContent';
+import { budgetSectionQuery } from '../graphql/queries';
 
-export const budgetSectionQuery = gqlV2/* GraphQL */ `
+export const budgetSectionQueryWithHost = gqlV2/* GraphQL */ `
```
Comment (znarf) [inline components/collective-page/sections/Budget.js:29]:
```suggestion
export const budgetSectionWithHostQuery = gqlV2/* GraphQL */ `
```

File: pages/expenses.js [hunk 3]
```diff
@@ -356,11 +356,11 @@ const expensesPageQuery = gqlV2/* GraphQL */ `
           name
           slug
           type
-          supportedPayoutMethods
           settings
           plan {
             id
           }
+          ...ExpenseHostFields
```
Comment (znarf) [inline pages/expenses.js:363]: If you're adding this, remove the fields that are duplicates.

Comment (SudharakaP) [reply pages/expenses.js:355]: Thanks for the review. Done. 👍🏽

**Comment by SudharakaP** [comment]
@znarf : Thanks a bunch for the review. I've corrected those issues. 😃

**Review by Betree — COMMENTED** [review]
This is probably breaking the cache set up in https://github.com/opencollective/opencollective-api/blob/d0f7ce9b3a8f48424764eb5312e936275c84f294/server/graphql/cache.js#L58. We should investigate to make sure this stays cached.

**Pull request merged**